py3status/modules/netdata.py

```diff
@@ -28,8 +28,8 @@
 
 SAMPLE OUTPUT
 [
-    {'full_text': 'eth0'},
-    {'color': '#FF0000', 'full_text': 'LAN(Kb):  16.9↓   5.2↑', 'separator': True},
+    {'full_text': 'eth0 '},
```

Comment:
We merged everything into one format for `netdata.py` not long ago. Here, we replace separator with a space. The `keyboard_locks.py` typo change in this PR also contain trailing spaces.